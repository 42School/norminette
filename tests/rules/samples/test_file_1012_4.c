t_montype	g_globale;
t_montype	g_glob;