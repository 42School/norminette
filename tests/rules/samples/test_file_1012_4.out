[36mtest_file_1012_4.c[0m - [32mIsVarDeclaration[0m In "GlobalScope" from "None" line 1":
		<IDENTIFIER=t_montype> <TAB> <IDENTIFIER=g_globale> <SEMI_COLON> <NEWLINE>
[36mtest_file_1012_4.c[0m - [32mIsVarDeclaration[0m In "GlobalScope" from "None" line 2":
		<IDENTIFIER=t_montype> <TAB> <IDENTIFIER=g_glob> <SEMI_COLON> 
test_file_1012_4.c: Error!
Notice: GLOBAL_VAR_DETECTED  (line:   1, col:   1):	Global variable present in file. Make sure it is a reasonable choice.
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
Notice: GLOBAL_VAR_DETECTED  (line:   2, col:   1):	Global variable present in file. Make sure it is a reasonable choice.
